### ISSUE TITLE
Daemon GetInfo API can't deserialize responses

### ISSUE THREAD

**Issue by dimalinux** [MarinX/monerorpc#9]
The `GetInfoResponse` struct has a mistyped `BusySyncing` field. It should be `bool,` but it's typed `string`.
https://github.com/MarinX/monerorpc/blob/master/daemon/models.go#L200-L201

Here's test code snippet:
```
endpoint := fmt.Sprintf("http://%s:%d/json_rpc", monerodHost, monerodPort)
daemonCli := monerorpc.New(endpoint, nil).Daemon
info, err := daemonCli.GetInfo()
```
The last line above returns an error with the message:
```
json: cannot unmarshal bool into Go struct field GetInfoResponse.busy_syncing of type string
```
I'm using monerod release v0.18.1.1 with go1.18.5.

**Comment by MarinX** [comment]
Can confirm that [GetInfoResponse](https://www.getmonero.org/resources/developer-guides/daemon-rpc.html#get_info) struct has changed.

Working on PR.

**Comment by MarinX** [comment]
Merged and released as v1.0.2

**Comment by dimalinux** [comment]
Thanks! It's working good now.
